lib/teleterm/api/proto/teleterm/v1/access_request.proto

```diff
@@ -14,11 +14,11 @@
 
 syntax = "proto3";
 
-package teleport.terminal.v1;
+package teleterm.v1;
```

Comment:
"teleport.teleterm.v1" seems to be the better path for me. See the top-level comment about unifying paths even further.

Same for prehog.